README.md

```diff
@@ -26,7 +26,7 @@ Therefore, a database containing all the necessary passwords is accessible.
 
 ### Accessing the passwords database
 
-- Install [KeePassX](https://www.keepassx.org/downloads)
+- Install [KeePassXC](https://keepassxc.org/download/#macos)
```

Comment:
Remove #macOS and let the website choose the OS?
```suggestion
- Install [KeePassXC](https://keepassxc.org/download)
```

Reply:
Of course, sorry for the copy-pasta!